module/documents/actor/actor.mjs

```diff
@@ -1779,18 +1779,25 @@ export default class Actor5e extends Actor {
    * carried by an Actor.
    * @returns {Promise<Actor5e>}
    */
+
   convertCurrency() {
     const curr = foundry.utils.deepClone(this.system.currency);
```

Comment:
Hm, I wonder if a deep clone is really needed here, or if a shallow clone would suffice.

Reply:
I only copied this from the previous convert currency function. I'm not too keen on the exact difference. Do you know why the original author may have used deep clone?

Reply:
I can't find a shallow clone. What would you suggest using here instead?

Reply:
Use `array.slice()` (or rather, `this.system.currency.slice`) with no arguments.

Reply:
Since `this.system.currency` isn't an array, the slice method won't work. If we want to avoid a deep clone, `const curr = this.system.currency;` does work as expected.